apps/docs/pages/guides/getting-started.mdx

```diff
@@ -84,6 +100,26 @@ export const meta = {
   })}
 </div>
 
+export const useCases = [
+  {
+    title: 'OpenAI Vector Search',
+    href: '/guides/getting-started/quickstarts/openai-vector-search',
+    description: `Build your own custom ChatGPT with Next.js, OpenAI and pg_vector.`,
+    icon: '/docs/img/integrations/logos/openai_logo.png',
```

Comment:
@supabase/frontend is there a way to make the `GlassPanel` work with pngs/jpegs, or will I need to transform these into svgs?

Reply:
You can pass jsx if you like, should still work.

Reply:
we should use svgs for these I think

Reply:
https://www.svgrepo.com/svg/306500/openai

Reply:
you can add an optional `icon_name-light.svg` in the same directory and it will use the lightmode version automatically